conductor-client/src/main/kotlin/com/openlattice/hazelcast/serializers/AclKeyStreamSerializer.kt

```diff
@@ -0,0 +1,46 @@
+package com.openlattice.hazelcast.serializers
+
+import com.hazelcast.nio.ObjectDataInput
+import com.hazelcast.nio.ObjectDataOutput
+import com.kryptnostic.rhizome.hazelcast.serializers.ListStreamSerializers
+import com.openlattice.authorization.AclKey
+import com.openlattice.hazelcast.StreamSerializerTypeIds
+import com.openlattice.mapstores.TestDataFactory
+import org.springframework.stereotype.Component
+
+@Component
+class AclKeyStreamSerializer : TestableSelfRegisteringStreamSerializer<AclKey> {
+
+    companion object {
+
+        @JvmStatic
+        fun serialize(out: ObjectDataOutput, `object`: AclKey) {
+            ListStreamSerializers.fastUUIDListSerialize(out, `object`)
+        }
+
+        @JvmStatic
+        fun deserialize(`in`: ObjectDataInput): AclKey {
+            return AclKey(*ListStreamSerializers.fastUUIDArrayDeserialize(`in`))
```

Comment:
* does an extra array creation when this AclKey() constructor is already defensive and will do its own copy. So would be better to use the `fastUUIDListDeserialize`